packages/react-composites/tests/browser/chat/app/FakeAdapterApp.tsx

```diff
@@ -27,6 +32,9 @@ export const FakeAdapterApp = (): JSX.Element => {
     verifyParamExists(params.fakeChatAdapterModel, 'fakeChatAdapterModel')
   ) as ChatAdapterModel;
 
+  // Optional params
+  const useFrLocale = Boolean(params.useFrLocale);
```

Comment:
So this is just telling the fake adapter to use the french locale for the tests? Do we need to run the tests against the other locales?

Reply:
Yes, for now. I just wanted to convert the old Localization UI test which used French for this PR.